iroh-gateway/Cargo.toml

```diff
@@ -51,6 +51,8 @@ toml = "0.5.9"
 http-serde = "1.1.0"
 config = "0.13.1"
 async-trait = "0.1.56"
+rust-crypto = "0.2.36"
```

Comment:
this is outdated, please use individual crates from https://github.com/rustcrypto